config/locales/de.yml

```diff
@@ -381,11 +382,11 @@ de:
     filter: "Hier kannst du die Suchergebnisse filtern:"
     success: "Wir haben"
     result: {
-      one: " eine Speakerin gefunden!",
-      other: " %{count} Speakerinnen gefunden! Yeah! "
+      one: " eine Speakerin gefunden! <i class='fa fa-caret-down' style='font-size: 21px'></i>",
+      other: " %{count} Speakerinnen gefunden! <i class='fa fa-caret-down' style='font-size: 21px'></i>"
     }
     more_options: "<b>Keine passende Speakerin gefunden?</b><br> Vielleicht helfen Dir diese Links weiter:"
-    search_categories_html: "Du kannst auch in <a href='%{root_path}/#categories_anchor'>Kategorien und Themen</a> stöbern."
+    search_categories_html: "Du kannst auch in <a href='%{root_path}/#categories_anchor'>Kategorien </a>und <a href='#tag_cloud_anchor'>Themen</a> stöbern."
```

Comment:
hier würde ich in "anderen Kategorien" schreiben. Also anderen ergänzen. Außerdem müsste hinter Kategorien vor dem closing `</a>` tag das Leerzeichen weg.